SU2_CFD/src/integration/CIntegration.cpp

```diff
@@ -146,6 +146,11 @@ void CIntegration::Space_Integration(CGeometry *geometry,
     }
   }
 
+  /*--- Fixed values for turbulence quantities. ---*/
+  if(config->GetTurb_Fixed_Values()){
+    solver_container[MainSolver]->Impose_Fixed_Values(geometry, config);
+  }
```

Comment:
I would like something more generic here, for example call the method by default and let it decide whether it does anything or not.

At some point I want to grab all the logic of calling the different boundary conditions and move it to a single method of the solvers, to reduce the verbosity of CSolver's interface.

Reply:
I made some changes in 5654648. Do you think one should give this an even more generic flavor, e.g. by renaming the function `Impose_Fixed_Values`?

Reply:
It is fine for now, in the future it can become part of the "BC method" I mentioned.